blocks/library/shortcode/preview.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { withAPIData, Spinner, SandBox } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { compose } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentPostId } from '../../../editor/store/selectors';
```

Comment:
We can't import functions from the "editor" module in the "blocks" module, this creates an issue with bundle size. since the editor module will be duplicated in that case.

I think it's fine here to avoid the selector and just do something like `state.currentPost.id`

Reply:
@niranjan-uma-shankar are you going to address this?

Reply:
@westonruter @youknowriad I've addressed this in the most recent commit, thanks.